backend/app/services/knowledge/summary_service.py

```
@@ -0,0 +1,782 @@
+                    if len(content) > MAX_DOCUMENT_CONTENT_LENGTH:
+                        logger.warning(
+                            f"[SummaryService] Document too long ({len(content)} chars), truncating"
+                        )
+                        content = (
+                            content[:MAX_DOCUMENT_CONTENT_LENGTH]
+                            + "\n\n[Content truncated...]"
+                        )
+
+                    return content
+
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)